include/ck_tile/ops/gemm/kernel/batched_gemm_kernel.hpp

```diff
@@ -63,12 +53,12 @@ struct BatchedGemmKernel
     using Kargs = BatchedGemmKargs;
     using Hargs = BatchedGemmHostArgs;
 
-    __host__ static constexpr auto GridSize(const Hargs& h)
+    __host__ static constexpr auto GridSize(const Hargs& k)
```

Comment:
Can we use `Kargs `instead of `Hard `and `kargs `instead of `kargs `?

Reply:
I think we should keep separation between `Hargs` and `Kargs`. In general those two may be different - and that's the case in GroupedGEMM. `Hargs` should be used for user called host functions. Whereas `Kargs` are our internal version of arguments which would be provided to device kernel entry point (`operator()`). Apart from above I'm also in favor of using more self-explanatory variable names.

Reply:
I kept them separate but refactored a little bit so it makes more sense